nsis/SectionCoreReqs.nsh

```diff
@@ -53,20 +53,13 @@
     ; in order to run (including CDVD!) -- and really there should be more but we don't have working
     ; SPU2 null plugins right now.
  
-    File ..\bin\Plugins\GSnull.dll
+    ;File ..\bin\Plugins\GSnull.dll
```

Comment:
I only removed the plugins that we got rid of and weren't required for functionality. 

I could also keep the plugins in the installer by simply adding another section (unchecked by default) for null plugins and experimental plugins like the new release of USBqemu?